methods.py

```diff
@@ -50,7 +50,7 @@ def disable_warnings(self):
 
 def force_optimization_on_debug(self):
     # 'self' is the environment
-    if self["target"] != "template-release":
+    if self["target"] == "template-release":
```

Comment:
Is this really meant to be `"template-release"` with a dash and not `"template_release"` with an underscore?

Reply:
All that chat discussion about != and == made me miss the even more obvious error.